unity/XRTProject/Assets/Scripts/Training/TrainingManager.cs

```diff
@@ -82,6 +84,8 @@ private void Awake()
             {
                 CurrentUser = user;
                 OnUserReady?.Invoke();
+                Logger = gameObject.AddComponent<Logger>();
+                Logger.Setup(apiService, TrainingModule, user);
```

Comment:
Best way to attach it to TrainingManager? Or should create an independent component?

Reply:
Probably better for you to make the Logger to be its own singleton independent from the training module. Then the training manager would access logger to setup.

Reply:
Feedback from Agid. Make Logger a singleton and have TrainingManager set it up.